lib/src/screens/send/parse_address_from_domain.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:cake_wallet/entities/openalias_record.dart';
+import 'package:cake_wallet/entities/unstoppable_domain_address.dart';
+import 'package:cake_wallet/src/widgets/alert_with_one_action.dart';
+import 'package:cake_wallet/utils/show_pop_up.dart';
+import 'package:flutter/material.dart';
+import 'package:cake_wallet/generated/i18n.dart';
+
+const topLevelDomain = 'crypto';
+
+Future<String> parseAddressFromDomain(
+    BuildContext context, String domain, String ticker) async {
```

Comment:
This function is overloaded it's parsing domain and presenting popup, so need to separate those functionality into each own function. Move "logic" part from "screens" directory.